services/src/main/scala/cromwell/services/metadata/impl/WriteMetadataActor.scala

```diff
@@ -55,6 +63,43 @@ class WriteMetadataActor(override val batchSize: Int,
     dbAction.map(_ => allPutEvents.size)
   }
 
+  private def sanitizeInputs(
+    metadataWriteActions: NonEmptyVector[MetadataWriteAction],
+    metadataKeysToClean: List[String]
+  ): NonEmptyVector[MetadataWriteAction] =
+    metadataWriteActions.map { metadataWriteAction =>
+      val metadataEvents =
+        metadataWriteAction.events.map { event =>
+          if (
+            metadataKeysToClean.contains(event.key.key)
+            || (event.key.key.contains(":")
+              && metadataKeysToClean.contains(event.key.key.substring(0, event.key.key.indexOf(":"))))
+          ) {
+            event.value match {
+              case Some(_) =>
+                val value = event.value.get
```

Comment:
We try to avoid this kind of unprotected `get` - we know in our heads when writing the code that it's safe because we just confirmed that `event.value` is `Some(_)`, but the compiler doesn't know, and as the code evolves the guarantee can be lost. You can do something like this to make it compiler-checked (also added a `copy`, see other comment below):
```
event.value match {
    case Some(theVal) => 
        event.copy(value = Option(theVal.copy(value = StringUtil.cleanUtf8mb4(theVal.value)))
    case _ => event
}
```